src/bilder/components/concourse/steps.py

```diff
@@ -192,6 +192,47 @@ def _manage_worker_node_keys(
         )
 
 
+@deploy("Pull down pre-bundled resource types")
+def _install_resource_types(
+    concourse_config: ConcourseWorkerConfig, sudo=True, host=None, state=None
+):
+    if concourse_config.additional_resource_types:
```

Comment:
You can remove the `if` conditional and change the `for` loop to be:
`for resource in concourse_config.additional_resource_types or []:`